pkg/api/transaction_test.go

```diff
@@ -158,25 +166,29 @@ func TestTransactionList(t *testing.T) {
 			PendingTransactions: []api.TransactionInfo{
 				{
 					TransactionHash: txHash1,
-					Created:         time.Unix(storedTransactions[txHash1].Created, 0),
-					Data:            hexutil.Encode(storedTransactions[txHash1].Data),
 					To:              storedTransactions[txHash1].To,
+					Nonce:           storedTransactions[txHash1].Nonce,
 					GasPrice:        bigint.Wrap(storedTransactions[txHash1].GasPrice),
 					GasLimit:        storedTransactions[txHash1].GasLimit,
-					Value:           bigint.Wrap(storedTransactions[txHash1].Value),
-					Nonce:           storedTransactions[txHash1].Nonce,
+					GasTipCap:       bigint.Wrap(storedTransactions[txHash1].GasPrice),
+					GasFeeCap:       bigint.Wrap(storedTransactions[txHash1].GasPrice),
+					Data:            hexutil.Encode(storedTransactions[txHash1].Data),
+					Created:         time.Unix(storedTransactions[txHash1].Created, 0),
 					Description:     storedTransactions[txHash1].Description,
+					Value:           bigint.Wrap(storedTransactions[txHash1].Value),
 				},
 				{
 					TransactionHash: txHash2,
-					Created:         time.Unix(storedTransactions[txHash2].Created, 0),
-					Data:            hexutil.Encode(storedTransactions[txHash2].Data),
 					To:              storedTransactions[txHash2].To,
+					Nonce:           storedTransactions[txHash2].Nonce,
 					GasPrice:        bigint.Wrap(storedTransactions[txHash2].GasPrice),
 					GasLimit:        storedTransactions[txHash2].GasLimit,
-					Value:           bigint.Wrap(storedTransactions[txHash2].Value),
-					Nonce:           storedTransactions[txHash2].Nonce,
+					GasTipCap:       nil,
```

Comment:
why nil?

Reply:
because in above case we have already tried one with some amount so i thought maybe we should also see what happens with nil